### ISSUE TITLE
[Feature request]: "Best-of" system

### ISSUE THREAD

**Issue by nordowl** [Matistan/MinecraftDeathSwap#2]
As it currently is, a game is instantly won by a player, if the opponent dies (given that you're playing 1v1).

What if you could configure the number of deaths a player needs to lose/win.
For example, if player B dies, player A gets one point. The first player to reach a set amount of points wins.

This would make a game more long-lasting and interesting.

On top of that, players could optionally keep their inventories, so you don't have to farm your items anew (although this can easily be done by hand with a `gamerule` command).
Or, optionally, have the players' inventories reset after scoring a point and automatically give each player a set of wooden tools or even customized items.

Maybe it would also be interesting to teleport all players to a new random spot on the map after a point was scored. The issue is that this could cause a bit of lag on weaker servers.

**To summarize my ideas:**
- Point system, where player A gets a point if player B dies
- After scoring X points, a player wins
- Display current score to everyone after a death
- Respawn with full inventory or custom set of items
- Respawn somewhere else on the map

Again, these are only ideas. You don't have to implement any of this (I'm aware this might not be that easy)

**Comment by Matistan** [comment]
Oh my god, that's a lot of features, how do you come up with these haha. If I have time, maybe I will implement them in the future
